packages/eslint-plugin/src/rules/member-ordering.ts

```diff
@@ -251,6 +251,9 @@ function getNodeType(node: Member): string | null {
     case AST_NODE_TYPES.TSConstructSignatureDeclaration:
       return 'constructor';
     case AST_NODE_TYPES.TSAbstractClassProperty:
+      return node.type && functionExpressions.includes(node.type)
+        ? 'method'
+        : 'field';
```

Comment:
this will never return `method` because `node.type` will always be `TSAbstractClassProperty` which will never be in the `functionExpressions` array.
```suggestion
      return 'field';
```